media/js/firefox/new.js

```diff
@@ -171,12 +171,39 @@
                     $thankYou.focus();
                 }, 500);
             }
+
+            // Added measurements fire only when the download is fired
+            spring_campaign_measurement();
         }
 
         function show_scene_anim(scene) {
             show_scene(scene, true);
         }
 
+        // Pixel to be removed on July 31st, 2015 (Bug 1168440)
+        function spring_campaign_measurement () {
+            var _dntStatus = navigator.doNotTrack || navigator.msDoNotTrack;
+            var fxMatch = navigator.userAgent.match(/Firefox\/(\d+)/);
```

Comment:
We could (should?) always track IE since we can't know the users' choice since it's "enabled" by default.